app/views/visits/_form.html.erb

```diff
@@ -37,8 +41,8 @@
         </div>
       <% end %>
 
-      <%= form.date_field :next_visit_on,
-                          help: "Format: DD/MM/YYYY", min: '2018-01-01', max: '2019-01-01', value: visit.next_visit_on || '2018-01-01' %>
+      <%= form.text_field :next_visit_on, label: "Next visit date", autofocus: true, placeholder: "DD/MM/YYYY",
+                          help: "Format: DD/MM/YYYY", value: visit.next_visit_on %>
```

Comment:
- Same as above, why not `date_field`?
- `autofocus` shouldn't probably be true here.

Reply:
Good catch; I'll remove autofocus.